pkg/aws_config_server/list_roles.go

```diff
@@ -56,17 +56,11 @@ func listRoles(ctx context.Context, svc iamiface.IAMAPI) ([]*iam.Role, error) {
 			return !lastPage
 		},
 	)
-	if aerr, ok := err.(awserr.Error); ok {
-		if aerr.Code() == "403" { // access denied error
-			logrus.Error(err)
-			return output, nil
-		}
-	}
 	if err != nil {
-		return nil, errors.Wrap(err, "unable to get listRolesOutput")
+		logrus.Error(err)
```

Comment:
I feel like ignoring all errors here is kind of dangerous. We really only want to ignore errors we explicitly decide to.